src/DetailPage.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import request from 'superagent';
+import Loading from './Loading.js';
+import './detail.css';
+
+export default class DetailPage extends Component {
+    state = {
+        selectedItem: {},
+        loading: false,
+    }
+
+    componentDidMount = async () => {
+        this.setState({ loading: true });
+
+        const data = await request.get(`https://pokedex-alchemy.herokuapp.com/api/pokedex?pokemon=${this.props.match.params.pokemonName}`);
+
+        await this.setState({
+            selectedItem: data.body.results.find(item => item.pokemon === this.props.match.params.pokemonName),
+            loading: false
+        });
+    }
+
+    render() {
+        return (
+            <div className='detail-body'>
+                <h3>DETAIL PAGE:</h3>
+
+                <div>
+                    {this.state.loading ? <Loading /> :
+                        <div className='detail-block' style={{ backgroundColor: this.state.selectedItem.color_1, }}>
```

Comment:
Nice!! Love how you used the selected pokemon item color from the API to style the background!